src/conn/conn_cache_pool.c

```diff
@@ -456,19 +471,37 @@ __cache_pool_assess(WT_SESSION_IMPL *session, uint64_t *phighest)
 		new = cache->bytes_read;
 		/* Handle wrapping of eviction requests. */
 		if (new >= cache->cp_saved_read)
-			cache->cp_current_read = new - cache->cp_saved_read;
+			cache->cp_pass_read = new - cache->cp_saved_read;
 		else
-			cache->cp_current_read = new;
+			cache->cp_pass_read = new;
 		cache->cp_saved_read = new;
-		if (cache->cp_current_read > highest)
-			highest = cache->cp_current_read;
+
+		/* Update the application eviction count information */
+		new = cache->cp_saved_app_evicts;
```

Comment:
I'd avoid the use of C++ operators as C variable names.  It will bite us at some point.